### ISSUE TITLE
Add custom delimiter to Import-DbaCsv

### ISSUE THREAD

**Issue by jvillegas74** [sqlcollaborative/dbatools#6488]
### Summary of new feature
I will be great to be able to have be able to use custom delimiters in CSV files. 
* Clear/concise description of what the problem is that the new feature can solve.
I have a process that need to import big TXT file with a custom (2-char) delimiter ^!
### Proposed technical details (if applicable)
-Delimeter ^!
* e.g. Blog post reference that shows example code or functionality

### Latest version of dbatools as of writing
Latest
* Please provide the latest released version of dbatools module.

**Comment by wsmelton** [comment]
This is restricted by what the LumenWorks library allows...

**Comment by potatoqualitee** [comment]
Yeah, Lumenworks only allows a char as seen here: https://github.com/phatcher/CsvReader/blob/master/code/LumenWorks.Framework.IO/Csv/CsvReader.cs#L36

I'll have to rewrite the whole command with a new library to accommodate a lot of our requests, it's on my agenda but it's pretty tough to do, so it may be a bit.

**Comment by potatoqualitee** [comment]
@claude please add support for a custom delimiter. check out lumenworks' support and integrate it.
